Microsoft.Toolkit.Services/Services/Internal/UWpPasswordManager.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if WINRT
```

Comment:
I recommend moving files that are UWP specific to a new UWP folder and using wildcards in the csproj to only include them for uap10.0 (similar to the other services)